game/scripts/npc/abilities/sandking_epicenter.txt

```diff
@@ -52,7 +52,7 @@
       "03"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "epicenter_damage"                                "110 110 110 220 440"
+        "epicenter_damage"                                "110 130 150 300 500" //OAA
```

Comment:
I'd probably also nerf some of other Sand King's spells (like sandstorm aoe maybe) to combat these pretty big buffs to his ulti.

Reply:
Sand King's ult is his worst spell. That's why it needs a buff.

Reply:
yes and literally everything else about sand king is overpowerd. a 2k range stun, a 500 huge aoe damaging spell + invis and the ability to explode early creep camps with phase boots. Nerfs to those spells are needed if we want to buff SK ult's

Reply:
Yes he has one of the best stuns in the game. However his damage output drops off heavily. Castic finale is fine how it is, that ability is designed to allow SK to farm fast. Sandstorm is good but its not OP.